lib/controller.rb

```diff
@@ -4,7 +4,22 @@ def initialize(game)
   end
 
   def play
-    @game.play_move until @game.is_over?
-    @game.tie_or_won
+    play_game = true
+    while play_game
+      @game.play_move until @game.is_over?
+      @game.tie_or_won
+      play_game = play_again?
+    end
+    @game.display.show_exit_message
+  end
+
+  def play_again?
```

Comment:
The game is now definitely breaching SRP! All the bits you've done are right, but I think there should be more objects than just game :). What's clear is that @display doesn't fit within @game.

Reply:
I agree, display doesn't fit within Game. I'll change this. Thanks!